ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/logging/arjunaI18NLogger.java

```diff
@@ -1566,6 +1566,11 @@ public void warn_objectstore_JDBCImple_over_max_image_size(int imageSize,
     @Message(id = 12391, value = "Could not initialize object store '{0}' of type '{1}'", format = MESSAGE_FORMAT)
     public String init_StoreManager_instantiate_class_failure(String name, String type);
 
+    @Message(id = 12392, value = "AbstractRecord.create {0} failed to create record class {1}.",
+           format = MESSAGE_FORMAT)
+    @LogMessage(level = WARN)
+    public void warn_coordinator_AbstractRecord_create_failure(Integer arg1, String arg2);
```

Comment:
Maybe it would be better to name the arguments with the `type` and the `className` rather then using arg1 and arg2?

Reply:
This is not a valid objection. The parameter names follows the general style of the other methods in the file. We have agreed in the past to following the style that was already present when making changes.

Reply:
Ah, yes I see. Looking at the diff it shows the one above does have it but that style seems to be in the minority. I suspect as this file is not really re-used too much due to the nature of its purpose. Thanks for looking @mmusgrov